common/scripts/update-npm-dependencies.js

```diff
@@ -0,0 +1,141 @@
+const ncu = require('npm-check-updates');
+const Octokit = require('@octokit/rest');
+const { installAndRun, findRushJsonFolder, RUSH_JSON_FILENAME } = require("./install-run");
```

Comment:
The double quotes hint at a lack of lint. Perhaps it would be worth it to move the script into a project and have the common `lint` framework, also benefit from having it's dependencies.

Reply:
new project in this repo or in a new repo?